map:
  cell_size: 0.4 #m
  map_width: 49
  map_height: 49
  robot_offset: 0 # offset between robot and the center of the map on the x axis
  rotation: true #rotation of the map 
  lambda_max: 1.0e+5 #max value of lambda
  lambda_unmeasured: -1 #default value of lambda in unmeasured cells
  min_measure_nbr: 1 #minimum number of measures on a cell to get a lambda value
  measure_duration: 1.0 #s duration a hit/miss is stored in a cell
  publish_rate: 10 #Hz
sensor:
  radius_filter: 1.0 #m filter upon the radius of the laser scan  
  angular_filter: 3.1415 #rad symetric filter upon the angular boundaries of the laser scan
  error_region_x: 0.4 #m
  error_region_y: 0.4 #m
  aperture_angle: 1.483e-3 #rad
  ph: 0.99
  pm: 0.9999
tf:
  odom: 'odom'
  lidar: 'velo_link'
  robot: 'base_link'
normals_estimation:
  neighbor_radius: 20 #neighbors
  max_radius: 2.0 #meters
  min_neighbors: 3 #neighbors
visibility:
  min_range: 1.0 #m
  max_range: 5.0 #m
  probability: 0.5 # visibility is the distance at which a lidar beam reaches an object with this probability

